scripts/test/watch

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-
-set -e
-
-filewatcher \
-    -L 5 \
-    -x '**/*.swp' \
-    -x .git \
-    -x build \
-    -x .idea \
-    -- \
-    sh -c 'go test -timeout 10s -v ./${dir} || ( echo; echo; exit 1 )'
```

Comment:
The Makefile is only being used to define developer operations. This is a developer operation. Maybe it doesn't fit in the Makefile, but that sounds like an argument against using a Makefile for this purpose.